### ISSUE TITLE
Unused variable failure string for unused import should be more specific

### ISSUE THREAD

**Issue by adidahiya** [palantir/tslint#550]
The following code reports `unused variable`, when it _could_ be more helpful and say `unused import`:

```ts
import * as csp from "js-csp";
```
